### ISSUE TITLE
Allow to configure the base href or something

### ISSUE THREAD

**Issue by davsclaus** [hawtio/hawtio-java#3]
I could not get v2 to work as /hawtio etc. The index.html seems fixed to / as context-path.

```
 <head>
    <title>openshift-jvm</title>
    <base href="/">
    <meta charset="UTF8">
```

I think we should have some way of configuring this dynamic - not sure how this should be done.

@gashcrumb any thoughts?

**Comment by gashcrumb** [comment]
Yeah, so in general we'll need a solution for this so that's it's not a pita to put consoles in different contexts.  Really the best way to do it is to have the server set it on the fly when it serves index.html.  I've been meaning to come up with a way to do this in hawtio-node-backend to start off with, as it's been a thorn in my side in the past :-)

**Comment by gashcrumb** [comment]
For now we can set 'base' in the index.html in hawtio-console-assembly to be /hawtio, it'll require adjusting the config for the gulp 'connect' task so it's served out at /hawtio also.
